Index: M_incl
===================================================================
--- M_incl	(revision 224)
+++ M_incl	(working copy)
@@ -1,6 +1,6 @@
 # set default platform
 ifndef platform
-  platform = LINUX_gcc
+  platform = LINUX_64_gcc
 endif
 
 # set default debug
@@ -18,7 +18,7 @@
 # include makefile defaults
 include ${UTILS_DIR}/makefile_includes/Makefile.defaults
 
-RPM = 0
+RPM = 1
 ifeq ($(RPM), 1)
   OPTS+=-fPIC 
 endif
@@ -54,13 +54,13 @@
 VGUI_LIB = -lvgui
 
 #MOC	 = /usr/lib/qt4/bin/moc
-MOC	 = /usr/share/qt4/bin/moc
+#MOC	 = /usr/share/qt4/bin/moc
 #MOC = /sw/lib/qt4-x11/bin/moc
-#MOC      = /usr/lib64/qt4/bin/moc
+MOC      = /usr/lib64/qt4/bin/moc
 
 #QT4DIR = /usr/lib/qt4
-QT4DIR = /usr/share/qt4
-#QT4DIR = /usr/
+#QT4DIR = /usr/share/qt4
+QT4DIR = /usr/
 #QT4DIR = /sw/lib/qt4-x11/
 
 QT_INC = -I$(QT4DIR)/include -I$(QT4DIR)/include/QtGui -I$(QT4DIR)/include/QtCore -I$(QT4DIR)/include/QtOpenGL -I$(QT4DIR)/include/Qt3Support -I$(QT4DIR)/mkspecs/default -DQT3_SUPPORT
